scripts/ci/upload_artifact.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+import paramiko
+import sys
+from io import StringIO
+
+def upload_files(remote_host, remote_user, remote_port, ssh_private_key, source, target):
+    privkey = StringIO(ssh_private_key)
+    ssh_key = paramiko.Ed25519Key.from_private_key(privkey)
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+    try:
+        ssh.connect(remote_host, port=remote_port, username=remote_user, pkey=ssh_key)
+        
+        sftp = ssh.open_sftp()
+        for root, dirs, files in os.walk(source):
+            relative_dir_path = os.path.relpath(root, source)
+            remote_dir_path = os.path.join(target, relative_dir_path).replace("\\", "/")
+            try:
+                sftp.mkdir(remote_dir_path)
+            except OSError:
+                pass  # Directory probably already exists
+
```

Comment:
An exception shouldn't happen since we are outside the for loop :/
I think it's safe to not guard it anymore.

Reply:
It's still possible (e.g., https://sdk.devbuilds.komodo.earth/artifact-uploader-ci-script directory already exists for this branch)

ref: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2158#discussion_r1668622099

Reply:
Oops, right, I missed direct files of the `root`.